docs/3_2_model_exchange_api.adoc

```diff
@@ -76,7 +76,7 @@ _[This allows limited simulation backward in time._
 _As soon as an event occurs (<<fmi3EnterEventMode>> was called), going back in time is impossible, because <<fmi3EnterEventMode>> / <<fmi3UpdateDiscreteStates>> can only compute the next discrete state, not the previous one.]_
 
 Function <<get-and-set-variable-values,`fmi3Set{VariableType}`>>::
-Only for variables with <<causality>> = <<input>> and <<variability>> = <<continuous>>.
+Only for variables with <<causality>> = <<input>> and <<variability>> = <<continuous>>, and <<state,continuous-time states>>.
```

Comment:
```suggestion
Only for variables with <<variability>> = <<continuous>> and either <<causality>> = <<input>> or <<state,continuous-time states>>.
```

Reply:
After discussion with Torsten, I agree that the original proposal with ", and <<state,continuous-time states>>."  is better English